docs/config.rst

```diff
@@ -84,7 +84,7 @@ useful inside shared application containers),
 
 ::
 
-    http://public:private@host:port/1?raven.http.proxy.host=proxy.example.com&raven.http.proxy.port=8080
+    http://public:private@host:port/1?sentry.http.proxy.host=proxy.example.com&sentry.http.proxy.port=8080
```

Comment:
Prefixing these with `sentry.*` feels even more redundant now than it did before. Is there a reason not to remove it?